examplescripts/ubuntu_deploy.py

```diff
@@ -1,130 +1,271 @@
+"""
+This script is for deploying two ubuntu containers and testing sshing between them
+Steps:
+- creating new pool
+- loading network with name `management` to create containers on it or creating new network with same name if it doesn't exist
+- deploy network with access node
+- deploy two containers on the network
+- use ssh forwarding to ssh between the containers
+"""
+
 from jumpscale.loader import j
 from time import sleep
-import uuid
-import random
+import netaddr
+from jumpscale.clients.explorer.models import NextAction, State, WorkloadType
+
+
+zos = j.sals.zos.get()
+WALLET_NAME = "test"
+network_name = "management"
+ip_range = "10.100.0.0/16"
+wg_quick = None
+
+
+bad_nodes = [
```

Comment:
is this still needed?

Reply:
there are still some nodes that fail because of the storage issue "not enough capacity in pool of type ssd" and there is one node that is not responding to any workload